Dockerfile

```diff
@@ -135,7 +135,7 @@ RUN cd /opt/bsim && \
 	chmod ag+w . -R
 
 RUN wget ${WGET_ARGS} https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v${ZSDK_VERSION}/zephyr-sdk-${ZSDK_VERSION}-linux-x86_64-setup.run && \
-	sh "zephyr-sdk-${ZSDK_VERSION}-linux-x86_64-setup.run" --quiet -- -d /opt/toolchains/zephyr-sdk-${ZSDK_VERSION} && \
```

Comment:
why remove --quiet?

Reply:
Arguably so that we can have better build log with all the details about the Zephyr SDK installation; but, looking at the existing build logs, it looks we already get that even with `--quiet` (because that only applies to the SFX portion).

I will add back `--quiet`.